udata_recommendations/templates/dataset-recommendations.html

```diff
@@ -1,12 +1,12 @@
 {%- if reco_reuses %}
-<div id="dataset-recommendations" class="hidden-xs">
+<div id="dataset-recommendations" class="fr-displayed-sm">
```

Comment:
I find it weird that we don't show it on small screens, should we keep it as is?

Reply:
I did that ✋ 😉 The goal was to preserve screen real estate when on mobile, without investing into a "condensed" view for mobile.

If we find a satisfactory solution we can make it work on mobile.

Reply:
Thank you for the input! I guess it can be refactored later on based on UX findings.
In my opinion though, showing recommended reuse is particularly useful on mobile for many users.

Reply:
Following our experimentations about accordions for search filters, we can also try them here. What do you think ?